.husky/pre-push

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+. "$(dirname "$0")/common.sh"
+
+echo "Fixing any lint errors before commit!"
+yarn lint-fix
```

Comment:
Super minor, but this might be a good use-case of lint-staged + `npx`? See [stylelint's](https://github.com/stylelint/stylelint/blob/master/.husky/pre-commit).

(also, would we want to make this a pre-commit versus a pre-push hook?)

Reply:
sound good to me!! and pre-commit makes sense